File: Orion/Controllers/Requests/Relation/RelationFilterParams.cs
using Orion.Infrastructure.Queries.Filtering;

namespace Orion.Controllers.Requests.Relation
{
    public class RelationFilterParams
    {
        public FilterParam<Guid>[]? Id { get; set; }
        public FilterParam<string>? Name { get; set; }
        public FilterParam<string>? VatNumber { get; set; }
        public FilterParam<string>? Street { get; set; }
        public FilterParam<string>? PostalCode { get; set; }
        public FilterParam<string>? City { get; set; }
        public FilterParam<string>? Country { get; set; }
        public FilterParam<string>? Language { get; set; }
        public FilterParam<string>? Email { get; set; }
        public FilterParam<bool>? IsCompany { get; set; }
    }
}

File: Orion/ClientApp/src/app/shared/requests/relation/relation-filter.params.ts
import {FilterParam} from "../../infrastructure/queries/filter/filter-param";


export class RelationFilterParams {
  id?: FilterParam<number[]>;
  name?: FilterParam<string>;
  vatNumber?: FilterParam<string>;
  street?: FilterParam<string>;
  postalCode?: FilterParam<string>;
  city?: FilterParam<string>;
  country?: FilterParam<string>;
  language?: FilterParam<string>;
  email?: FilterParam<string>;
  isCompany?: FilterParam<boolean>;
}
